src/components/Tree/styled/itemContainerLi.js

```diff
@@ -2,19 +2,28 @@ import styled from 'styled-components';
 
 const ItemContainerLi = styled.li`
     box-sizing: border-box;
-    margin-left: 28px;
-    ${props =>
-        props.hasChildren &&
-        `
-        margin-left: 0px;
-    `};
-
     &:focus {
         outline: 0;
     }
-
     &:focus > div {
-        border-left: #01b6f5 solid 4px;
+        background-color: ${props => props.theme.rainbow.palette.action.active};
+    }
+    ${props =>
+        props.hasChildren &&
+        `
+        > div > button {
+            visibility: visible;
+        }
+    `};
+    > div {
+        margin-left: ${props =>
```

Comment:
I think it is better to apply styles directly to the div, which is more maintainable and readable